core/utils/xml.js

```diff
@@ -16,65 +16,71 @@
  * @name Blockly.utils.xml
  * @namespace
  */
-goog.provide('Blockly.utils.xml');
+goog.module('Blockly.utils.xml');
+goog.module.declareLegacyNamespace();
 
 
 /**
  * Namespace for Blockly's XML.
  */
-Blockly.utils.xml.NAME_SPACE = 'https://developers.google.com/blockly/xml';
+const NAME_SPACE = 'https://developers.google.com/blockly/xml';
+exports.NAME_SPACE = NAME_SPACE;
 
 /**
  * Get the document object.  This method is overridden in the Node.js build of
  * Blockly. See gulpfile.js, package-blockly-node task.
+ *
+ * Note that this function is named myDocument so as to not shadow the
+ * global of the same name, but (for now) exported as .document to not
+ * break existing importers.
```

Comment:
Three things:
  1) myDocument -> getDocument
  2) Does this break [node packaging](https://github.com/google/blockly/blob/64188ae27297b1ca23a3244cc987dfbfc6e98f34/scripts/package/node/core.js#L25)?
  3) Do you want to export `getDocument` as well?

Reply:
This renaming _should_ have been entirely internal and totally fine from the point of view of any users of this package (and therefore should not have been able to break the node packaging), _except that I modified `createElement` and `createTextNode` to call `getDocument()` instead of `exports.document()`, making it impossible to monkey-patch these calls_.  Ooops.

I want to fix the money patching properly but for now I will just modify these two internal calls to call `exports.document` so that everything works the way it previously did.

Reply:
1. Fixed.
2. See previous comment.
3. Yes, but not in this PR.  In the next PR I'll export `getDocument` and also a `setDocument`, and use the latter instead of monkey-patching.